10-inference.Rmd

```diff
@@ -0,0 +1,806 @@
+in general, the population is the complete collection of individuals or cases we are interested in studying. 
+Further, in the above question, we are interested in computing a quantity&mdash;the proportion
+of iPhone owners&mdash;based on the entire population. This is our **population parameter**:
+in general, a population parameter is a numerical characteristic
+of the entire population. In order to compute this number in the example above, we would need to ask 
+every single undergraduate in North America whether or not they own an iPhone. In practice,
+directly computing population parameters is often time-consuming and costly, and sometimes impossible. 
+
+A more practical approach would be to collect measurements for a **sample**: a subset of
+individuals collected from the population. We can then compute a **sample statistic**&mdash;a numerical
+characteristic of the sample&mdash;that estimates the population parameter. For example, if we 
+randomly selected 100 undergraduate students across North America (the sample) and computed the fraction of those
+students who own an iPhone (the sample statistic), we might suspect that that fraction is a reasonable
+estimate of the full population fraction. 
+
+<center>
+![Figure 11.1: Population versus sample](img/population_vs_sample.svg){}
+</center>
+
+Note that proportions are not the *only* kind of population parameter we might be interested in.
+Let's consider another example question that we might tackle with statistical inference:
+
+*What is the average price-per-night of one-bedroom apartment rentals in Vancouver, Canada?*
+
+Here, the population consists of all one-bedroom apartment rental offerings in Vancouver, and the population
+parameter is the *average price-per-night*. But even within this one example, we could also be interested
+in many other population parameters: the median price, the fraction of 
+one-bedroom apartments that cost more than \$200 per night,
+the standard deviation of the price, and the list goes on.
+If we were somehow able to observe the whole population of one-bedroom apartments in Vancouver, 
+we could compute each of these numbers exactly; therefore these are all population parameters. 
+There are many kinds of observations and population parameters that you will run into in practice,
+but in this chapter we will focus on two settings:
+
+1. Using categorical observations to estimate the proportion of each category
+2. Using quantitative observations to estimate the average (or mean)
+
+## Sampling distributions
+
+### Sampling distributions for proportions
+
+Let's start with an illustrative (and tasty!) example. Timbits are 
+bite-sized doughnuts sold at Tim Hortons, a popular Canadian-based fast-food restaurant
+chain founded in Hamilton, Ontario, Canada. 
+
+![Timbits. Source: wikimedia.org](https://upload.wikimedia.org/wikipedia/commons/thumb/7/75/Timbits2.jpg/1600px-Timbits2.jpg){width=50%} 
```

Comment:
I think we can create our own timbits photo 😉  I volunteer for this job 😋 !

Reply:
Thanks, Tiffany! I will leave this for you 👍